src/components/Pages/Admin/View.vue

```diff
@@ -216,16 +234,57 @@ export default {
 
                 this.GeogebraViews = new GeogebraViews(this.groupsInClass, {
                     log: this.$log,
-                    width: this.$refs.ibox_content.clientWidth - 60,
+                    width: 400, // this.$refs.ibox_content.clientWidth - 60,
```

Comment:
Why do you hardcode those values? Isnt 400x400 small?

Reply:
Client requested smaller tiles
https://makimo.atlassian.net/browse/MATHNETNG-105